common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -79,6 +79,7 @@ def do_my_expensive_update(course_key):
     4. The thing that listens for the signal lives in process, but should do
        almost no work. Its main job is to kick off the celery task that will
        do the actual work.
+    5. The name of your handler function's parameters *must* be "sender" and "course_key"
```

Comment:
"name" => "names"

Also, you might as well add that the `kwargs` argument should be present.